lib/parse_identifier_pmid.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'parse_identifier'
+
+# Parse the PMID identifiers in a PublicationIdentifier
+class ParseIdentifierPMID < ParseIdentifier
+
+  URI_PREFIX = 'https://www.ncbi.nlm.nih.gov/pubmed/'.freeze
+
+  # Does the data validate?
+  def valid?
+    value.present? && uri.present?
+  end
+
+  # Extract a PMID value
+  # @return [String|nil]
+  def value
+    @value ||= extract_value
+  end
+
+  private
+
+    # Compose a URI
+    # @return [String|nil]
+    def compose_uri
+      return nil if value.blank? || URI_PREFIX.blank?
```

Comment:
How can a constant you've already defined be blank?

Reply:
This could be a copy-paste mistake, thanks.  Specs did not expose it because it's a no-op, as you say.  This can be easily fixed.

Reply:
Removed.